### DISCUSSION TITLE
what directory do I run npm start on?

### DISCUSSION THREAD

**Discussion by alfredopastasauce** [benawad/dogehouse#246] [Q&A] [answered]
I just cloned my fork onto my dektop and I just want to know if i need to get any packages and also what directory do I run npm on?

thanks everyone

**Comment by PoseidonCoder** [comment, marked as answer]
I suggest looking [here](https://github.com/benawad/dogehouse#how-to-run-locally) to learn how to run it. If you want to run the frontend then you will want to navigate to kofta and type `npm i` to install all of the dependencies. Then you will need to type `npm start` to run it.

**Comment by mweser** [comment]
Thanks, @PoseidonCoder, very helpful 🙂
